include/wlr/types/wlr_text_input.h

```diff
@@ -0,0 +1,35 @@
+#ifndef WLR_TYPES_WLR_TEXT_INPUT_H
+#define WLR_TYPES_WLR_TEXT_INPUT_H
+
+#include <wayland-server.h>
+#include <wlr/types/wlr_surface.h>
+
+
```

Comment:
Spacing nits: no newline between `#define` and `#include`s, and only one space between `#include`s and declarations